### ISSUE TITLE
Error in mobilenetv2 preprocess_input

### ISSUE THREAD

**Issue by jiwoong-choi** [qubvel/segmentation_models#56]
First of all, thank you for sharing nice code. I found it very useful for one of my projects.

I found a problem in get_preprocessing for 'mobilenet' and 'mobilenetv2'
Simple code:
```python
from segmentation_models.backbones import get_preprocessing
import numpy as np
img = np.zeros((32, 32, 3))
preprocess_input = get_preprocessing('mobilenetv2') # or 'mobilenet' gives the same error
img = preprocess_input(img.astype(np.float32))
```
Error message:
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/path/to/segmentation_models/backbones/mobilenetv2.py", line 92, in preprocess_input
    return imagenet_utils.preprocess_input(x, mode='tf', **kwargs)
  File "/path/to/lib/python3.5/site-packages/keras_applications/imagenet_utils.py", line 186, in preprocess_input
    data_format = backend.image_data_format()
AttributeError: 'NoneType' object has no attribute 'image_data_format'
```

Note that no such error occured when I tried the same code with other backbone models like 'resnet18' or 'vgg19' etc. There were two ways to get rid of this error:
Method 1: import Classifiers from classification_models
```python
from classification_models import Classifiers
import numpy as np
img = np.zeros((32, 32, 3))
preprocess_input = Classifiers.get_preprocessing('mobilenetv2') # or 'mobilenet'
img = preprocess_input(img.astype(np.float32))
```
Method 2: specify backend keyword argument
```python
import keras
from segmentation_models.backbones import get_preprocessing
import numpy as np
img = np.zeros((32, 32, 3))
preprocess_input = get_preprocessing('mobilenetv2') # or 'mobilenet'
img = preprocess_input(img.astype(np.float32), backend=keras.backend)
```
So the original preprocessing function for mobilenet(v2) works fine but the updated one in [Line 15~16 in segmentation_models/backbones/\_\_init\_\_.py](https://github.com/qubvel/segmentation_models/blob/3a33a0661a8969ae6c67ea77c2163ba927e4bef9/segmentation_models/backbones/__init__.py#L15) is the problematic part.

Although not a big problem, it is still annoying as you have to treat the case for mobilenet(v2) separately when you write down a code for general backbones. I'll be more than happy if you consider fixing this issue. Thanks!

**Comment by qubvel** [comment]
Hi @jiwoong-choi 
```python
from segmentation_models.backbones import get_preprocessing
import numpy as np
img = np.zeros((32, 32, 3))
preprocess_input = get_preprocessing('mobilenetv2') # or 'mobilenet' gives the same error
img = preprocess_input(img.astype(np.float32))
```
This code works correctly for me :thinking: 
What is your keras and keras-applications versions?

**Comment by jiwoong-choi** [comment]
I tested it with

* python 3.5.1+
* keras 2.2.4
* keras-application 1.0.7
* segmentation-models 0.2.0
* tensorflow 1.10.0

I have tried the simple code above after installing segmentation-models via pip and by cloning into the repository, but the same error occurs every time. 
But after googling about this error message, it seems like a bug in keras itself. (I've got the same error when I tried the sample code in this [post](https://stackoverflow.com/questions/52717181/keras-create-mobilenet-v2-model-attributeerror))

**Comment by jiwoong-choi** [comment]
I have also tested with:
* keras 2.2.0
* keras-application 1.0.7
* segmentation-models 0.2.0
* tensorflow 1.10.0

and surprisingly the code worked without error. But one thing I noticed here is that I had to reinstall keras-applications 1.0.7 after installing keras 2.2.0, which is incompatible according to a warning from pip.
What I did:
```
pip install keras==2.2.0 # this installs keras-applications 1.0.2 automatically
pip uninstall keras-applications
pip install keras-applications==1.0.7 # after doing this the sample code worked just fine
```
warning message I got:
```
keras 2.2.0 has requirement keras-applications==1.0.2, but you'll have keras-applications 1.0.7 which is incompatible.
```

**Comment by qubvel** [comment]
Ok, thanks for reporting, I will fix it for a keras 2.2.4

**Comment by wave-transmitter** [comment]
Hello, has this been fixed?
